oeps/processes/oep-0055/decisions/0003-readme-specification.rst

```diff
@@ -0,0 +1,183 @@
+
+* **Getting Started** - A section to indicate how you can begin development on
+  the new component.
+
+* **Getting Help** - A section to indicate where and how the user can get help
+  with the component.  It should link to relevant docs as well as places where
+  the users can get help with from other humans.
+
+* **How to Contribute** - A section to indicate how new users can contribute to
+  this component.  This section should at the very least contain a link to the
+  general `How to Contribute <https://openedx.org/r/how-to-contribute>`_
+  documentation for the Open edX Project.
+
+* **Open edX Code of Conduct** - A section that links to the `Open edX Code of
+  Conduct <https://openedx.org/code-of-conduct/>`_
+
+* **People** - A section that tells people how they can find the current
+  maintainers of this component.  Usually a link to the Backstage page for the
+  component and to the catalog-info.yaml file.
+
+* **Reporting Security Issues** - A link to the e-mail address where security
+  issues can be reported.
+
+Template
+========
+
+.. code-block:: rst
+
+   ##########
+   Repo Name
+   ##########
+
+   | |License: AGPL v3| |Status| |Python CI| |Publish package to PyPi|
+
+   .. |License: AGPL v3| image:: https://img.shields.io/badge/License-AGPL_v3-blue.svg
+     :target: https://www.gnu.org/licenses/agpl-3.0
+
+   .. |Python CI| image:: https://github.com/openedx/DoneXBlock/actions/workflows/ci.yml/badge.svg
+     :target: https://github.com/openedx/DoneXBlock/actions/workflows/ci.yml
+
+   .. |Publish package to PyPi| image:: https://github.com/openedx/DoneXBlock/actions/workflows/pypi-release.yml/badge.svg
+     :target: https://github.com/openedx/DoneXBlock/actions/workflows/pypi-release.yml
+
+   .. |Status| image:: https://img.shields.io/badge/status-maintained-31c653
+
+   Purpose
+   *******
+
+   A few sentences that tell you what this code does and when you might want to use it.
+
+   Getting Started
+   ***************
+
+	Developing
+	==========
+
+	How can a new developer get started with developing on this component?  Is
+	there a short set of commands they could run? Is it complicated and this should
+	be a link out to a published larger How-To?
+
+   .. code:: bash
+
+   	  git clone git@github.com:openedx/the_repo.git
+   	  virtualenv venv/DoneXBlock/
+   	  source venv/DoneXBlock/activate
+   	  make upgrade
+   	  make install
+   	  make dev.run
+
+   You can interact with the DoneXBlock in the Workbench by navigating to http://localhost:8000
+
+
+	Deploying
+	=========
+
+	How can a new user, go about deploying this component? Is it just a few commands? Is there a larger how-to that should be linked here?
+
+	For details on how to deploy this component, checkout the `deployment how-to`_
+
+	.. _deployment how-to: https://docs.openedx.org/projects/this-project/how-tos/how-to-deploy-this-component.html
+
+
+   Getting Help
+   ============
```

Comment:
Is this meant to be a subsection of "Getting Started"? Reminder to ensure that the template is in sync with the specification above. Same question for many of the sections below. I'm guessing the heading-level is just off, but it is unclear.